src/Trixi.jl

```diff
@@ -30,7 +30,7 @@ using Reexport: @reexport
 # MPI needs to be imported before HDF5 to be able to use parallel HDF5
 # as long as HDF5.jl uses Requires.jl to enable parallel HDF5 with MPI
 using MPI: MPI
-
+using OrdinaryDiffEq
```

Comment:
Same here. Do you need to load OrdinaryDiffEq?

Reply:
The same like previous